package.json

```diff
@@ -15,6 +15,7 @@
   "author": "Fran Mendez <fmvilas@gmail.com> (fmvilas.com)",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@asyncapi/raml-dt-schema-parser": "^1.0.4",
```

Comment:
why do we still test the RAML parser here and not https://github.com/asyncapi/raml-dt-schema-parser/ ?

Reply:
I actually don't mind it being testing here, great to have some integration tests. I guess we could actually duplicate this test in the RAML parser as well?

Reply:
maybe as a short term solution, but ideally there should be a dummy parser there in the repo used for testing that also could be used as a reference in docs. So we at least need to have a followup issue for that to improve

Reply:
:thinking: Yea you are right, would probably be best to use a dummy parser here and then in the RAML parser add this test :)

Reply:
Good point. Moving the test.

Reply:
@derberg @jonaslagoni https://github.com/asyncapi/raml-dt-schema-parser/pull/2

Reply:
Ok, finished moving tests (for OpenAPI too).